src/updateIds.js

```diff
@@ -143,22 +144,21 @@ const parseSuite = suiteTitle => {
 const replaceSuiteTitle = (title, replace, content) => {
   const lines = content.split('\n');
 
-  // try to find string near keyword
-  for (const lineNumber in lines) {
-    const line = lines[lineNumber];
-    for (const keyword of SUITE_KEYWORDS) {
-      if (line.match(keyword)) {
-        for (let i = lineNumber; i < lines.length; i++) {
-          if (lines[i].includes(title)) {
-            lines[i] = line.replace(title, replace);
-            return lines.join('\n');
-          }
+  // try to find string near keyword & exclude import lines
+  const updatedLines = lines.map(line => {
+    if (!line.match(LINE_START_REGEX)) {
```

Comment:
```suggestion
   // ignore lines with kewords
    if (line.match(LINE_START_REGEX)) return;
```

let's simplify it a bit

Reply:
You are right!... has been fixed